src/BloomExe/XmlHtmlConverter.cs

```diff
@@ -132,6 +123,29 @@ public static XmlDocument GetXmlDomFromHtml(string content, bool includeXmlDecla
 			return dom;
 		}
 
+		/// <summary>
+		/// Tity is over-zealous. This is work-around. After running Tidy, then call RemoveFillerInEmptyElements() on the same text
+		/// </summary>
+		/// <returns></returns>
+		private static string AddFillerToKeepTidyFromRemovingEmptyElements(string content)
+		{
+			//hack. tidy deletes <span data-libray='somethingImportant'></span>
+			// and also (sometimes...apparently only the first child in a parent) <i some-important-attributes></i>
+			content = content.Replace("></span>", ">REMOVEME</span>");
+			content = content.Replace("></i>", ">REMOVEME</i>");
```

Comment:
You haven't changed this, only moved it...but I'd be surprised if we don't need this for bold and underline also.